Content.Server/Physics/Controllers/ChaoticJumpSystem.cs

```diff
@@ -0,0 +1,74 @@
+using System.Linq;
+using Content.Server.Physics.Components;
+using Robust.Shared.Random;
+using Robust.Shared.Timing;
+using Robust.Shared.Physics.Systems;
+using Robust.Shared.Physics;
+using System.Numerics;
+
+namespace Content.Server.Physics.Controllers;
+
+/// <summary>
+/// A component which makes its entity periodically chaotic jumps arounds
+/// </summary>
+public sealed class ChaoticJumpSystem : EntitySystem
+{
```

Comment:
this and `ChasingWalkSystem` should inherit `VirtualController` and have their logic in `UpdateBeforeSolve` instead of `Update` to work better with physics, see how `RandomWalkSystem` does it